src/views/GovernanceTaskList/AdditionalRequestInfo.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Alert, IconArrowForward } from '@trussworks/react-uswds';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Divider from 'components/shared/Divider';
+import { GetGovernanceTaskList_systemIntake as SystemIntake } from 'queries/types/GetGovernanceTaskList';
+import { RequestRelationType } from 'types/graphql-global-types';
+
+function AdditionalRequestInfo({ ...system }: SystemIntake) {
+  const { t } = useTranslation('itGov');
+
```

Comment:
I wonder if this component could belong in a shared component folder for use with TRB.  In https://github.com/CMSgov/easi-app/pull/2461 `/src/components/SystemCard` the data and use was identical for displaying system information.  The component is specific to a system, rather than IT Gov.  For the props, can probably literally define the type for each prop that needs to be rendered, rather than the generated query type.  Just a thought.

Reply:
Looking into this next. I was almost anticipating it, thanks for pointing it out here.